packages/generator-one-app-module/generators/app/index.js

```diff
@@ -63,8 +69,40 @@ module.exports = class extends Generator {
       type: 'input',
       name: 'moduleName',
       validate: helper.validateIfInputIsValidOrNot,
-      message: 'What is the name of your Module?',
-    }];
+      message: 'Name of your Module:',
+      store: false,
+    },
+    {
+      type: 'list',
+      name: 'moduleType',
+      default: 'root module',
+      message:
+        'Is this a root module or child module?',
```

Comment:
Can we add a link to what a root module or a child module is? That way the user can better inform themselves on what type of module they should generate.

Reply:
Added bit.ly shortened URLs, but I think it just makes the generator look messy. Do we really need to link to docs for each individual prompt? The general link to one app docs in below the logo, and likely when someone is using the generator they're following the quickstart, which'll have all pertinent info.